scripts/buildtools/check_tests.py

```diff
@@ -0,0 +1,116 @@
+print("===== STARTED RUNNING check_tests.py =====\n")
+
+import json
+import sys
+import argparse
+
+# Arguments parsing / help menu
+parser = argparse.ArgumentParser(description='Verify test results for skipped tests and tests with multiple passes.')
```

Comment:
How can skipped tests results be verified? 
If it means they will run but ignored, then it should be ignored tests and not skipped tests.

Reply:
That might confuse people even more, since go test/go testsum specifically show them as SKIPPED

Reply:
I think we can phrase it differently, like "Check test results for intentionally and unintentionally skipped tests, as well as tests that ran multiple times"